client/scenes/stream_tracking.cpp

```diff
@@ -243,6 +244,12 @@ void scenes::stream::tracking()
 						t.resume();
 					}
 
+					if (application::get_fb_face_tracking2_supported())
+					{
+						application::get_fb_face_tracker2().get_weights(t0 + Δt, &fb_face2);
```

Comment:
Either we do the extrapolation + latest (Δt = 0 and Δt = prediction) and server side needs to use a history system to store/retrieve the values, or we only query one time.

What I understand from current code is that it will get 2 samples per loop, which might get reordered, then server only stores one.

Reply:
We only get 1 sample and store that. But maybe extrapolating on the server side would make sense.

Reply:
`get_weights` is inside the `for (XrDuration Δt = 0; Δt <= prediction; Δt += std::max<XrDuration>(1, prediction))` loop, it will be executed twice

Reply:
so i guess the battery check also needs to be moved out?